extension/js/methods.js

```diff
@@ -24,60 +24,65 @@ This function lists down all the updates and requires a user confirmation.
 function update_packages(project) {
     $('#updatebtn').unbind();
     $('#updatebtn').click(function () {
-        var selectedPackages = common.get_selected_packages();
-        var html = $('<p> The following packages are about to be updated: </p> </br>');
-        $('#updatebtn').toggleClass('fa-wrench fa-spinner').addClass('fa-spin');
-        api.check_update(project, function (resp) {
-            $('#updatebtn').toggleClass('fa-wrench fa-spinner').removeClass('fa-spin');
-            var updates = resp.updates;
-            var packages = [];
+        if ($('#updatebtn').hasClass('fa-wrench')) {
```

Comment:
can you explain what you're doing here?

Reply:
This is to prevent the button from being clicked twice and invoking consequent requests before getting a response.

Reply:
you can always disable the button after the user clicks on it